More Survival Tools Series 5

chaosvolt opened the pull request:
It's that time again, it seems. Sticking with content updates, I've got
a shitload of MShock24 Modded tileset work I'm saving for next PR.
1. Removed the "plant fiber from withered plants" recipe, now that we have an official version using cattails/dogbane.
2. Added an override for the new plant fiber recipe, increasing output and requiring a distaff, while also adding cannabis (because hemp) to the list of usable plants.
3. Too lazy for cooked hot dogs? Added campfire hot dogs, for when you just want to roast some wieners over an open fire (childish giggling ensues at that statement).
4. Added cattaill stalks to tinder options.

chaosvolt commented:
I'm pondering the possibility of removing the "make plant fibers out of withered plants" idea entirely though, as for now it was merely a fix to unbreak mainline content. Likewise I might ponder removing the "dry cattails/dogbane into straw" idea in favor of only using the "flour byproduct" one..

Would hope to get opinions on both those ideas.

Coolthulhu commented:
How about: Withered plant (2x or more)->straw

chaosvolt commented:
Hmm. Maybe, though already added a recipe for turning cattail or dogbane into straw, so adding withered plants and bumping the requirement from 1 to 2 would be viable. Dunno if it's logical though, or if it's essential after adding straw as a by-product of flourmaking.

Coolthulhu commented:
More logical than fiber from dried plants or straw, though not essential at all as straw is quite useless.

As for flour straw - I'm pretty sure it already drops straw on harvest.

chaosvolt commented:
Oh derp, it does? I can presumably remove that then.

qbradq commented:
Strictly speaking, straw only comes from the stalks of grains. Not to be "that guy", just saying that's one more aspect to think about.

chaosvolt commented:
> Strictly speaking, straw only comes from the stalks of grains. Not to be "that guy", just saying that's one more aspect to think about.

Hence why I added it as a byproduct of making flour from wheat and related plants, and moved the non-cereal flour options into a separate recipe. I might remove the option to dry cattails as a result, but the reason I added that was due to the use of cattails and reeds as thatching or other straw alternatives.

As it is, we already have confusion between bales of hay yielding straw. o3o

qbradq commented:
Wow, that is confusion. Hay it's totally different than straw.

chaosvolt commented:
If I recall, it's only in the "bale of hay" terrain/furniture/something. Since either can be stored in bales, it's not that derpy.

So I might yank the "dry cattail into straw" idea, if getting it from making flour from grains is good enough?

chaosvolt commented:
While I'm at it, and other recent mainline items that might affect my mod? o3o

chaosvolt commented:
Meep. I'll take that as an "enough changes for the time being" I guess. o3o

chaosvolt commented:
And thank you for the merge. ^^

chaosvolt commented:
I'd kinda still wanted input on any further suggestions, plus feedback on the idea of crafting a tatami mat though. owo

Coolthulhu commented:
Not sure if tatami mat really has any use at the moment.
You could balance some existing items and recipes. For example, add batch crafting bonus to food/material crafting ones like drying hides.
Hobo stove is better than a hotplate, which doesn't really make sense.

chaosvolt commented:
> Not sure if tatami mat really has any use at the moment.

I think the main reason I thought of adding that is because it has some comfort bonus compared to sleeping on the ground, if I remember right. So a straw-based alternative to a rollmat would have a useful niche.

As for batch modding, I likely should. And hobo stove...hmm. Any suggestions on balancing it? Maybe increased fuel usage?

qbradq commented:
I don't see anything wrong with a fuel burning stove being better at cooking than a hot plate. That fits with my perceptions of reality :)

Coolthulhu commented:
It's not a gas stove, but a tinder stove.

You can't increase fuel usage (technical limitations). Unless you mean cost and volume of fuel - that would work. You could increase the volume of the stove to about 10 or so to compensate for its high resource efficiency.
While the low maximum fuel on the stove is probably rather annoying, it has a positive effect of another nerf: limiting batch sizes. Not sure if it's worth the annoyance, though.

chaosvolt commented:
Hmm. Might beef up the volume I guess. Tinder also currently has no volume, so I could give it volume to fix that. Maybe 1 volume per 100 tinder? Basically 2 stacks.

I already reduced the fuel cap from 100 to 50 in Series 3. >.o

Coolthulhu commented:
1 volume per 100 tinder would make it as volume-efficient as batteries, which are way rarer and more valuable. Even 1 volume per 25 wouldn't make it bad, because it is essentially free.

No need to reduce the fuel cap further. To make it matter, it would need a cap of about 10, which would make it incredibly unwieldy to use. With increased volume, it could get increased fuel cap to compensate: 10 volume and 100 charges would be OK.

Each item needs its own role. Here hobo stove replaces hotplate pretty much entirely, which is a problem. With 10 volume, it would occupy a different niche - it would be more of a replacement for campfire rather than a portable mess kit.

chaosvolt commented:
Hmm, 10 volume for the hobo stove. It is assumed to be made of a tin can though. Then again, the real-life ideal can for a hobo stove is a COFFEE can, which is generally larger than a soup can.

As for the tinder volume, should I take it down to 1 volume per 50, maybe? owo

qbradq commented:
If it's supposed to be made of a coffee can, then I think 10 max fuel make sense. I also like how it makes the stove fit into the other potable cooking options.

chaosvolt commented:
10 max fuel might be a tad low though. 25 might be a bit more reasonable.

Coolthulhu commented:
Well, if it's supposed to be a single can, then it should probably have some other limitation.
Limiting its fuel would make sense from the realism point of view, but it doesn't actually balance it, while making it way more annoying to use.

I'd balance it by making tinder more annoying to carry around. If the tinder had quite a bit of volume itself, it wouldn't offer that much advantage over starting a fire and using that. 1 volume per 25 charges, for example.
After all, tinder probably isn't exactly an amazing fuel.

chaosvolt commented:
I see. In that case I might leave the max capacity of the stove untouched, and instead bump up the stove's volume and give tinder some volume.

Coolthulhu commented:
Recipe uses pretty much free stuff, so changing how much it produces isn't much. Volume is way more important here.

At 4 with 100 fuel, stove is better than hotplate. You could keep the 50 fuel capacity and just bump tinder volume, so that holding 100 charges would require 4 volume for loaded stove and 2 volume for extra tinder.

chaosvolt commented:
Ah I see, so keep the hobo stove's volume? Hmm. I do like the idea of the hobo stove being more bulky than a hotplate though. Hotplates are generally nice and flat, while a coffee can is...well, not. o3o

I might bump it up to 6 or 8 volume though.

chaosvolt commented:
Derp ensues. https://github.com/CleverRaven/Cataclysm-DDA/pull/12405
